code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -539,6 +539,17 @@
 	damage = 0
 	stamina = 40
 
+//pirate version
+/obj/item/gun/magic/hook/pirate
+	name = "boarding hook"
+	desc = "A long metal chain, with a heavy hook connected to the end that can make deadly blows in close quarters. Once used to board ancient merchant vessels, now used to drag unsuspecting victims into close range."
+	ammo_type = /obj/item/ammo_casing/magic/hook/bounty
+
+/obj/item/gun/magic/hook/pirate/melee_attack_chain(mob/user, atom/target, params)
+	. = ..()
+	user.changeNext_move(CLICK_CD_MELEE)//makes the hook unquick to swing
+	return .
+  
```

Comment:
```suggestion
/obj/item/gun/magic/hook/pirate
	name = "boarding hook"
	desc = "A long metal chain, with a heavy hook connected to the end that can make deadly blows in close quarters. Once used to board ancient merchant vessels, now used to drag unsuspecting victims into close range."
	ammo_type = /obj/item/ammo_casing/magic/hook/bounty

/obj/item/gun/magic/hook/pirate/melee_attack_chain(mob/user, atom/target, params)
	. = ..()
	user.changeNext_move(CLICK_CD_MELEE)//makes the hook unquick to swing
	return .
 
```

Reply:
Something still seems to be failing...

🥲